hr_holidays_legal_leave/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+HR Holidays Legal Leave
+=======================
+
+Allows the define which holiday type is to be considered legal/annual leave type.
+Currently Odoo assumes by default leave type with limit=False. This is a problem if
```

Comment:
what do you mean here :

Currently Odoo assumes by default leave type with limit=False